migrations/20170622165827-ports-migration.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
```

Comment:
Migrations are a part of Sequelize. You export the functions `up` and `down`. A migration is either `up`grading the database or `down`grading the database. 

[Sequelize Migrations](http://docs.sequelizejs.com/manual/tutorial/migrations.html)